elodie/filesystem.py

```diff
@@ -378,12 +389,22 @@ def process_file(self, _file, destination, media, **kwargs):
         if(move is True):
             stat = os.stat(_file)
             shutil.move(_file, dest_path)
+            if(os.path.exists(_file + '_original')):
+                os.remove(_file + '_original')
             os.utime(dest_path, (stat.st_atime, stat.st_mtime))
         else:
-            compatability._copyfile(_file, dest_path)
+            # exiftool preserves the original file with '_original' appended to
+            # the file name.
+            if(os.path.exists(_file + '_original')):
+                shutil.move(_file, dest_path)
+                shutil.move(_file + '_original', _file)
+            else:
+                compatability._copyfile(_file, dest_path)
```

Comment:
I'm not sure I follow this section of code. Could you update the comment to be a bit more detailed (instead of explaining it in this comment 😀)?